config/file/file_test.go

```diff
@@ -114,11 +114,14 @@ func testWatchFile(t *testing.T, path string) {
 	if err != nil {
 		t.Error(err)
 	}
-	defer f.Close()
```

Comment:
```go
defer func() {
        if err = f.Close(); err != nil {
		t.Fatalf("close file error(%v)", err)
	}
}()
```

Reply:
Is this better？

Reply:
On Windows systems, if you modify a file and call `watch.Next()` before closing it, the call may block indefinitely. This is because the `fsnotify` package uses the `syscall.GetQueuedCompletionStatus` function, which might not return the correct result until the file handle is closed.